resources/local/config.toml

```diff
@@ -252,6 +262,44 @@ blocklist_retain_duration = '1 minute'
 # ca_certificate = "ca_cert.pem"
 
 
+# ============================================
+# Configuration options for Connection Manager
```

Comment:
Should either be "the connection manager" or "connection management" ?

Reply:
Updated in 6a891781a543224cfcc8c5f875575bf06cd00c77